playwright/fixtures/index.ts

```diff
@@ -158,6 +171,27 @@ export const test = baseTest.extend<Options & Fixtures>({
 
     await use({ name: sliceName });
   },
+  repeatableZoneSlice: async ({ sliceLibrary, manager, createMock }, use) => {
+    const variation = createMock.model.sharedSliceVariation({
+      itemsFields: {
+        existing_field: createMock.model.richText(),
+      },
+    });
+    const model = createMock.model.sharedSlice({ variations: [variation] });
+
+    await manager.slices.createSlice({ libraryID: sliceLibrary.id, model });
+
+    await use({ name: model.name });
+  },
+  sliceLibrary: async ({ manager }, use) => {
```

Comment:
I think `firstSliceLibrary` (or `defaultSliceLibrary`?) works better than a `getSliceLibrary` function. We will most likely only use this value in fixtures to generate data, in which case we will only be using the first slice library.

If we are manipulating the UI, we should be using the actual UI instead to read values (I think, you tell me if that's incorrect).

I'll rename it, but let me know if you think otherwise.

Reply:
I like `firstSliceLibrary` easy and devs will clearly understand what they are using 🙂